storm-server/src/main/java/org/apache/storm/daemon/supervisor/BasicContainer.java

```diff
@@ -568,14 +570,7 @@ private int getMemOffHeap(WorkerResources resources) {
     }
 
     protected String javaCmd(String cmd) {
```

Comment:
Is there any need to keep this method now that it is available in Utils?

Reply:
When I took this out, a bunch of test classes started failing. They were using a Mock class to override this class method to return just "java". And in the test, they check the returned array. In order to avoid changing a whole bunch of other classes, I left this signature unchanged. But this definitely looks ugly and the tests should be fixed.

Reply:
Sounds good.  I'm fine keeping this for now.